src/components/ui/Checkbox/checkbox.scss

```diff
@@ -31,74 +70,74 @@
   }
 
   &:hover {
-    background-color: var(--fs-color-primary-bkg-light);
-    border-color: var(--fs-color-text);
-    box-shadow: 0 0 0 var(--fs-border-width) var(--fs-border-color-active);
+    background-color: var(--fs-checkbox-bkg-color-hover);
+    border-color: var(--fs-checkbox-border-color-hover);
+    box-shadow: var(--fs-checkbox-shadow-hover);
   }
 
   &:checked {
-    background-color: var(--fs-color-primary-bkg);
-    border-color: var(--fs-border-color-active);
+    background-color: var(--fs-checkbox-checked-bkg-color);
+    border-color: var(--fs-checkbox-checked-border-color);
     border-width: 0;
 
     &::before {
       opacity: 1;
     }
 
     &:hover {
-      background-color: var(--fs-color-primary-bkg-hover);
+      background-color: var(--fs-checkbox-checked-bkg-color-hover);
       box-shadow: none;
     }
   }
 
-  &[data-store-checkbox-partial="true"] {
-    background-color: var(--fs-color-neutral-0);
-    border: var(--fs-border-width) solid var(--fs-color-primary-bkg);
+  &:disabled {
+    cursor: not-allowed;
+    background-color: var(--fs-checkbox-disabled-bkg-color);
+    border: var(--fs-checkbox-disabled-border-width) solid var(--fs-checkbox-disabled-border-color);
+
+    &::before {
+      border-color: var(--fs-checkbox-disabled-border-color);
+    }
+
+    &:hover {
+      background-color: var(--fs-checkbox-disabled-bkg-color);
+      border: var(--fs-checkbox-disabled-border);
+
+      &::before {
+        border-color: var(--fs-checkbox-disabled-border-color);
+      }
+    }
+
+    + label {
+      color: var(--fs-checkbox-disabled-text-color);
+    }
+  }
+
+  &[data-fs-checkbox-partial="true"] {
+    background-color: var(--fs-checkbox-partial-bkg-color);
```

Comment:
Can you please also check when partial & disabled are true? The bg color should be the disabled one.

![preview-partial-disabled](http://g.recordit.co/azC6xPjbto.gif)